lib/ls.js

```diff
@@ -95,7 +95,7 @@ var lsFromTree = ls.fromTree = function (dir, physicalTree, args, silent, cb) {
   }
   console.log(out)
 
-  if (args.length && !data._found) process.exitCode = 1
+  if (args.length && !data._found && npm.command !== 'unbuild') process.exitCode = 1
```

Comment:
I'd like a comment here explaining why we're doing this, because I just know that when we go to rewrite things to not have a global npm object, that this bit is going to be super confusing. =p